pkg/config/config.go

```diff
@@ -302,6 +302,7 @@ func initConfig(config Config) {
 	// Cloud Foundry
 	config.BindEnvAndSetDefault("cloud_foundry", false)
 	config.BindEnvAndSetDefault("bosh_id", "")
+	config.BindEnvAndSetDefault("cf_os_hostname_aliasing", false)
```

Comment:
should we add some additional documentation to the config template regarding this option and what it entails?

Reply:
I don't think we need to/should. This option will be specific to the bosh agent, and will be configured via the tile, where it is documented (see the two PRs referencing this one).
The `cloud_foundry` and `bosh_id` options which are not documented in the template either.